README.md

```diff
@@ -896,50 +896,78 @@ turbo price --value 1024 --type bytes
+turbo revoke-credits --wallet-file ../path/to/my/wallet
 ```
 
-##### `list-approvals`
+##### `list-shares`
 
-List all given and received delegated payment approvals from the connected wallet or the provided native address.
+List all given and received credit share approvals from the connected wallet or the provided native address.
 
 Command Options:
 
-- `-a, --address <nativeAddress>` - Native address to list delegated payment approvals for
+- `-a, --address <nativeAddress>` - Native address to list credit share approvals for
 
 e.g:
 
 ```shell
-turbo list-approvals --address 2cor...VUa --wallet-file ../path/to/my/wallet
+turbo list-shares --address 2cor...VUa --wallet-file ../path/to/my/wallet
 ```
 
+## Turbo Credit Sharing
+
+Users can share their purchased Credits with other user's wallets by creating Credit Share Approvals. These approvals are created by uploading a signed data item with tags indicating the recipient's wallet address, the amount of Credits to share, and an optional amount of seconds that the approval will expire in. The recipient can then use the shared Credits to pay for their own uploads to Turbo.
+
+Shared Credits cannot be re-shared by the recipient to other recipients. Only the owner of the Credits can share or revoke Credit Share Approvals. Credits that are shared to other wallets may not be used by the original owner of the Credits for sharing or uploading unless the Credit Share Approval is revoked or expired.
+
+Approvals can be revoked at any time by similarly uploading a signed data item with tags indicating the recipient's wallet address. This will remove all approvals and prevent the recipient from using the shared Credits. All unused Credits from expired or revoked approvals are returned to the original owner of the Credits.
+
+To use the shared Credits, recipient users must provide the wallet address of the user who shared the Credits with them in the `x-paid-by` HTTP header when uploading data. This tells Turbo services to look for and use Credit Share Approvals to pay for the upload before using the signer's balance.
+
+For user convenience, during upload the Turbo CLI will use any available Credit Share Approvals found for the connected wallet before using the signing wallet's balance. To instead ignore all Credit shares and only use the signer's balance, use the `--ignore-approvals` flag. To use the signer's balance first before using Credit shares, use the `--use-signer-balance-first` flag. The Turbo SDK layer does not provide this functionality and will always use the signer's balance unless `paidBy` is provided.
+
+The Turbo SDK provides the following methods to manage Credit Share Approvals:
+
+- `shareCredits`: Creates a Credit Share Approval for the specified wallet address and amount of Credits.
+- `revokeCredits`: Revokes all Credit Share Approvals for the specified wallet address.
+- `listShares`: Lists all Credit Share Approvals for the specified wallet address or connected wallet.
```

Comment:
Do we have no APIs that can list ALL of the credit share approvals provided by the original credit holder?

Reply:
if you mean approvals from single payer to a single signer, no not on the SDK level. the payment API does provide this for compatibility with the getApproval from ARx tooling. but this can be achieved by using the SDK and a simple filter: `listShares(signerAddress).receivedApprovals.filter(a => a === payerAddress)